locale/en/LC_MESSAGES/django.po

```diff
@@ -1382,7 +1382,7 @@ msgstr "New deadline"
 
 #: deviations/forms.py
 msgid "DEVIATION_NEW_DEADLINE_DATE_HELPTEXT"
-msgstr "New submission deadline in the future in format YYYY-MM-DD HH:MM."
+msgstr "New submission deadline in the future."
```

Comment:
```suggestion
msgstr "New submission deadline."
```
I don't think it is necessary to emphasize "future" here. If the user tries to enter an invalid date in the past, then the form validation should show a descriptive error.